### PR TITLE
ubi9

### PR DESCRIPTION
**Pull request by clcollins**
- Auto sync from upstream
- Update branch to master
- Use personal fork
- Upadate base image to ubi9-minimal

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,96 +1,70 @@
...
+      procps-ng \
+      python3 \
+      python3-pip \
+      rsync \
+      sshuttle \
+      tar \
+      vim-enhanced \
+      wget \
+      && microdnf clean all \
+      && rm -rf /var/yum/cache
```
Comment (rendhalver) [inline Dockerfile:44]: This may need that `\`.
The build succeeded using `--no-cache` without it so it may not be necessary.

Comment (clcollins) [reply Dockerfile]: It won't need the \.  It's the last on the line.

Comment (rendhalver) [reply Dockerfile]: That would explain why it built properly. :)

**Pull request merged**